datafusion/core/src/lib.rs

```diff
@@ -204,6 +204,7 @@
 /// DataFusion crate version
 pub const DATAFUSION_VERSION: &str = env!("CARGO_PKG_VERSION");
 
+extern crate core;
```

Comment:
Why is this necessary?

Reply:
It's not :) Not sure where it came from but removed now.